File: practice/src/org/acme/grabber/enums/Tuning.java
package org.acme.grabber.enums;

import java.util.Arrays;
import java.util.List;

import static org.acme.grabber.enums.Note.*;

public enum Tuning {

    STANDARD(Arrays.asList(E, B, G, D, A, E)),
    STANDARD_D(Arrays.asList(D, A, F, C, G, D)),
    DROP_D(Arrays.asList(E, B, G, D, A, D)),
    DROP_C_SHARP(Arrays.asList(Ds, As, Fs, Cs, Gs, Cs)),
    DROP_C(Arrays.asList(D, A, F, C, G, C));

    private List<Note> noteList;

    Tuning(List<Note> noteList) {
        this.noteList = noteList;
    }

    public List<Note> getNoteList() {
        return noteList;
    }
}

File: practice/src/org/acme/grabber/entities/Tuning.kt
package org.acme.grabber.entities

import org.acme.grabber.entities.Note.*

enum class Tuning(val displayName: String, val notes: List<Note>) {

    STANDARD("Standard (6 string)", listOf(E, B, G, D, A, E)),
    STANDARD_D("Standard D (6 string)", listOf(D, A, F, C, G, D)),
    DROP_D("Drop D (6 string)", listOf(E, B, G, D, A, D)),
    DROP_C_SHARP("Drop C# (6 string)", listOf(Ds, As, Fs, Cs, Gs, Cs)),
    DROP_C("Drop C (6 string)", listOf(D, A, F, C, G, C)),
    DROP_A_6("Drop A (6 string)", listOf(B, G, D, A, E, A))
}

